docs/markup-language/README_es.md

```diff
@@ -0,0 +1,1618 @@
+---
+meta:
+  - nombre: descripción
+    contenido: >-
+      El protocolo OpenLaw se basa en un lenguaje de marcado para convertir a
+      los acuerdos de lengua natural en objetos legibles por máquina con
+      variables y lógica pertinentes definidos dentro de un documento en
+      concreto.
+---
+
+# Lenguaje de marcado
+
+El protocolo OpenLaw se basa en un lenguaje de marcado para transformar a los acuerdos hechos en idiomas naturales en objetos procesables por máquina con variables y lógica pertinentes, definidos dentro de un documento específico (lo que llamamos un "modelo"). Los modelos se pueden agrupar en "acuerdos", lo cual permite que se crean y se gerencien transacciones enteras en una cadena de bloques.
```

Comment:
As per your comment above, do you want to use `cadena de bloques` here?

Reply:
I'll make them all 'blockchain' for the moment, but if this is something that the team feels on a branding/messaging level would be better to nativize as 'cadena de bloques', it's an easy switch.

Reply:
Cool.